src/app/contribution-graph/contribution-graph.component.html

```diff
@@ -1,10 +1,14 @@
-<div id="canvas">
-    <div *ngFor="let col of columns">
-      <div class="graph-row" *ngFor="let row of numRow(col)">
-        <div class="circle"></div>
+<div id="container">
+  <div id="dayColumn">
```

Comment:
When naming css variables, please use `day-column` instead of `dayColumn`. See [here](https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849) for more on naming conventions

Reply:
got it, thanks for the link, very informative :)